doto-backend/src/routes/task-route.js

```diff
@@ -3,15 +3,15 @@ const router = express.Router();
 const authenticateToken = require('../config/token-setup').authenticateToken
 const Task = require('../models/Task');
 
-//GET ALL task
-router.get('/get', authenticateToken, (req,res) => {
-    let tasks = Task.find({"user": req.user.email})
-    .then(tasks => res.status(200).json(tasks))
-    .catch(err => res.status(400).json('Error: ' + err));
```

Comment:
Have you tested if this change breaks the API?

Reply:
Heya, I just tested it and it still works. I removed the declaration of the variable since visual studio code said it wasn't used anywhere

Reply:
hey can you start the frontend and see if the tasks are still persisted

Reply:
Yeah, it still works